### PR TITLE
xunion => xud

### PR DESCRIPTION
**Pull request by kilrau**

### CODE REVIEW

**Comment by moshababo** [comment]
Added the migration to all the rest of the files I found.

@sangaman take a look.

File: bin/xudcli [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 /* eslint-disable no-console */
 
-const XUClient = require('../lib/xuclient/XUClient');
+const XudClient = require('../lib/xudclient/XudClient');
```
Comment (sangaman) [inline bin/xudcli]: I think we should still call this file xucli. "XUD" = "Exchange Union Daemon" and "xucli" = "Exchange Union Command Line Interface." This also matches the naming conventions of bitcoind and btcd for instance.

File: README.md [hunk 2]
```diff
@@ -2,11 +2,11 @@
 
 [Exchange Union](https://www.exchangeunion.com/) (XU) is a decentralized exchange layer built on the [Lightning](http://lightning.network/) and [Raiden](https://raiden.network/) networks to enable trustless and instant cryptocurrency swaps and order fulfillment between exchanges.
 
-This repo contains the early stages of an XU node implementation which encompasses the following components:
+This repo contains the early stages of the node implementation 'xud' (Exchange Union Daemon) which encompasses the following components:
 
 * Integration with [lnd](https://github.com/lightningnetwork/lnd) and [raiden](https://github.com/raiden-network/raiden) nodes.
 * Orderbook data stored in a local mysql/mariadb database.
-* Peer-to-peer networking with other XU nodes via TCP.
+* Peer-to-peer networking with other xuds via TCP.
```
Comment (sangaman) [inline README.md]: I think "XU nodes" sounds better in this context even with the name change. As an analogy, in the bitcoin world we would say "networking with other bitcoin nodes" and not "networking with other bitcoinds."

File: README.md [hunk 3]
```diff
@@ -2,11 +2,11 @@
 
 [Exchange Union](https://www.exchangeunion.com/) (XU) is a decentralized exchange layer built on the [Lightning](http://lightning.network/) and [Raiden](https://raiden.network/) networks to enable trustless and instant cryptocurrency swaps and order fulfillment between exchanges.
 
-This repo contains the early stages of an XU node implementation which encompasses the following components:
+This repo contains the early stages of the node implementation 'xud' (Exchange Union Daemon) which encompasses the following components:
```
Comment (sangaman) [inline README.md]: This repo contains the early stages of the Exchange Union Daemon ("xud")...

File: README.md [hunk 4]
```diff
@@ -37,24 +37,24 @@ XUnion uses [MySQL](https://www.mysql.com/) or [MariaDB](https://mariadb.org/).
 ## Command-Line Interface
 
 ```bash
-~/xunion/bin $ ./xucli placeorder --price 0.12 --quantity 5
+~/xud/bin $ ./xudcli placeorder --price 0.12 --quantity 5
```
Comment (sangaman) [inline README.md]: See my comment on `xudcli`, I think we should keep the name as `xucli`.

File: README.md [hunk 5]
```diff
@@ -37,24 +37,24 @@ XUnion uses [MySQL](https://www.mysql.com/) or [MariaDB](https://mariadb.org/).
...
 {"fieldCount":0,"affectedRows":1,"insertId":1,"serverStatus":2,"warningCount":0,"message":"","protocol41":true,"changedRows":0}
-~/xunion/bin $ ./xucli getorders
+~/xud/bin $ ./xudcli getorders
```
Comment (sangaman) [inline README.md]: xucli

File: README.md [hunk 6]
```diff
@@ -37,24 +37,24 @@ XUnion uses [MySQL](https://www.mysql.com/) or [MariaDB](https://mariadb.org/).
...
 {"bids":[{"price":0.12,"quantity":5}],"asks":[]}
 ```
 
 ## Configuration
 
-The configuration file uses [TOML](https://github.com/toml-lang/toml) and by default is located at  `~/.xunion/xunion.conf` on Linux or `AppData\Local\XUnion\xunion.conf` on Windows. Default settings which can be overridden are shown below.
+The configuration file uses [TOML](https://github.com/toml-lang/toml) and by default is located at  `~/.xud/xud.conf` on Linux or `AppData\Local\xud\xud.conf` on Windows. Default settings which can be overridden are shown below.
```
Comment (sangaman) [inline README.md]: To follow the convention for windows AppData folders, lets stick with `AppData\Local\Xud`.

File: bin/xudcli [hunk 7]
```diff
@@ -16,26 +16,26 @@ const argDefinitions = [
 ];
 const args = commandLineArgs(argDefinitions);
 
-const xuClient = new XUClient(args.rpcport);
+const xudClient = new XudClient(args.rpcport);
```
Comment (sangaman) [inline bin/xudcli]: We can keep this as it was before.

File: bin/xudcli [hunk 8]
```diff
@@ -16,26 +16,26 @@ const argDefinitions = [
...
 
 let promise;
 switch (args.call) {
   case 'getinfo':
-    promise = xuClient.getInfo();
+    promise = xudClient.getInfo();
```
Comment (sangaman) [inline bin/xudcli]: Keep as before.

File: bin/xudcli [hunk 9]
```diff
@@ -16,26 +16,26 @@ const argDefinitions = [
...
     break;
   case 'getorders':
-    promise = xuClient.getOrders();
+    promise = xudClient.getOrders();
```
Comment (sangaman) [inline bin/xudcli]: Keep as before.

File: bin/xudcli [hunk 10]
```diff
@@ -16,26 +16,26 @@ const argDefinitions = [
...
     break;
   case 'placeorder': {
     const order = {
       price: args.price,
       quantity: args.quantity,
     };
-    promise = xuClient.placeOrder(order);
+    promise = xudClient.placeOrder(order);
```
Comment (sangaman) [inline bin/xudcli]: Keep as before.

File: bin/xudcli [hunk 11]
```diff
@@ -16,26 +16,26 @@ const argDefinitions = [
...
     break;
   }
   case 'connect': {
-    promise = xuClient.connect(args.host, args.port);
+    promise = xudClient.connect(args.host, args.port);
```
Comment (sangaman) [inline bin/xudcli]: Keep as before.

File: lib/xudclient/XudClient.js [hunk 12]
```diff
@@ -1,7 +1,7 @@
 const http = require('http');
 const assert = require('assert');
 
-class XUClient {
+class XudClient {
```
Comment (sangaman) [inline lib/xudclient/XudClient.js]: I'd keep this as it was before. "Exchange Union Client" is what we should be going for, "Exchange Union Daemon Client" is overkill. This is in line with the naming scheme of bitcoind, btcd, and others.

File: lib/xudclient/XudClient.js [hunk 13]
```diff
@@ -80,4 +80,4 @@ class XUClient {
   }
 }
 
-module.exports = XUClient;
+module.exports = XudClient;
```
Comment (sangaman) [inline lib/xudclient/XudClient.js]: Keep as before.

File: package.json [hunk 14]
```diff
@@ -1,26 +1,26 @@
...
   "version": "1.0.0-alpha.1",
-  "description": "Exchange Union node",
-  "main": "lib/XUnion.js",
+  "description": "Exchange Union Daemon",
+  "main": "lib/Xud.js",
   "bin": {
-    "xunion": "./bin/xunion",
-    "xucli": "./bin/xucli"
+    "xud": "./bin/xud",
+    "xudcli": "./bin/xudcli"
```
Comment (sangaman) [inline package.json]: Keep xucli.

**Comment by moshababo** [comment]
on it

**Pull request merged**